### PR TITLE
Update da.js

### PR DESCRIPTION
**Pull request by Rosefalk**
JS says it can't find fields that exists.

### CODE REVIEW

**Comment by Rosefalk** [comment]
Removing lang="da" from html tag and loading the locale via JS removed all logged errors...

File: locales/da.js [hunk 1]
```diff
@@ -90,7 +104,9 @@ shipping_address:
 payment_informations:
 "Betalingsinformation",
 payment_informations_bill_me_later:
-"Betal senere",
+"Betaling ikke nødvendig",
+payment_informations_no_payment_required:
+"",
```
Comment (JeanSebTr) [inline locales/da.js]: Is there an error here?
From google translate, `Betal senere` seemed correct for Pay Later and the new translation seems to fit `payment_informations_no_payment_required`

```suggestion
"Betaling ikke nødvendig",
```

**Comment by JeanSebTr** [comment]
Hi there,

Sorry for the long delay here. I've added a small review but everything else sound good.

Thanks for the naming chance between `Planen` and `Abonnements` !

File: locales/da.js (hunk 1 above)
Comment (Rosefalk) [reply locales/da.js]: @JeanSebTr It does seem like there's an error. It should be as below :) does that help?

> payment_informations_bill_me_later:
> "Betal senere",
> payment_informations_no_payment_required:
> "Betaling ikke nødvendig",

Comment (JeanSebTr) [reply locales/da.js]: Great! Just wanted to be sure!

**Pull request merged**